components/ws-daemon/cmd/run.go

```diff
@@ -142,3 +145,13 @@ func grpcProbe(cfg baseserver.ServerConfiguration) func() error {
 		return fmt.Errorf("grpc service not ready")
 	}
 }
+
+// createLVMDevices creates LVM logical volume special files missing when we run inside a container.
+// Without this devices we cannot enforce disk quotas. In installations without LVM this is a NOOP.
+func createLVMDevices() {
+	cmd := exec.Command("/usr/sbin/vgmknodes")
```

Comment:
Is it safe to be called twice? I'm afraid of when restarting `ws-daemon`

Reply:
Yes.

Reply:
@aledbf Thanks for your insight :pray: